pkg/commands/os.go

```diff
@@ -53,18 +57,6 @@ func (c *OSCommand) RunCommand(command string) error {
 	return err
 }
 
-// RunDirectCommand wrapper around direct commands
-func (c *OSCommand) RunDirectCommand(command string) (string, error) {
```

Comment:
Though this method is not being used in the master branch right now, there are some PRs up that make use of it. The difference between this method and other 'run command' methods is that it takes a big string and just sends that to e.g. `bash -c`. RunCommand/RunCommandWithOutput will break up your string into separate arguments, whereas RunDirectCommand will just directly give the whole command string as a single argument.